contrib/download-frozen-image-v2.sh

```diff
@@ -317,10 +318,10 @@ while [ $# -gt 0 ]; do
 			mediaType="$(echo "$manifestJson" | jq --raw-output '.mediaType')"
 
 			case "$mediaType" in
-				application/vnd.docker.distribution.manifest.v2+json)
+				application/vnd.docker.distribution.manifest.v2+json|application/vnd.oci.image.manifest.v1+json)
 					handle_single_manifest_v2 "$manifestJson"
 					;;
-				application/vnd.docker.distribution.manifest.list.v2+json)
+				application/vnd.docker.distribution.manifest.list.v2+json|application/vnd.oci.image.index.v1+json)
```

Comment:
Same order swap (and whitespace so it's easier to read):

```suggestion
				application/vnd.oci.image.index.v1+json | application/vnd.docker.distribution.manifest.list.v2+json)
```